src/brevitas/graph/equalize.py

```diff
@@ -1519,7 +1555,25 @@ def find_module(self, model, regions: List, prefix=''):
         else:
             for name, module in model.named_children():
                 full_name = prefix + '.' + name if prefix != '' else name
-                self.find_module(module, regions, full_name)
+                self.find_module(module, regions, full_name, blacklist_layers)
+
+    def find_module_by_name(self, model: nn.Module, regions: List[Region], prefix: str = ''):
+        """
+        Iterate through the model looking at immediate children of every module to look for named modules.
+        This allows us to stop the search when we meet a top-level module that is supported.
+        """
+        if prefix in self.layers_to_expand:
```

Comment:
Implicit assumption is that we use this function to layers_to_expand.

Fair assumption as we use the other function (`find_module`) for the supported layerwise modules.
It can also become a new args for the function I guess, but not much value in that at the moment